docs/machine-learning/tutorials/object-detection-model-builder.md

```diff
@@ -255,6 +255,21 @@ Two projects are created as a result of the training process.
     Top: 89.453415, Left: 481.95343, Right: 724.8073, Bottom: 388.32385, Label: Stop-Sign, Score: 0.99539465
     ```
 
+    > [!NOTE]
+    > The bounding box coordinates are normalized for a width of 800 pixels and height of 600 pixels. To scale the coordinates for your image, you need to:
+    >
+    >    1. Multiply top and bottom coordinates by the original image height, and multiply left and right coordinates by the original image width.
+    >    1. Divide top and bottom coordinates by 600, and divide left and right coordinates by 800.
+    >
+    >    For example, given the original image dimensions,`actualImageHeight` and `actualImageWidth`, and a prediction `prediction` with bounding box coordinates, the following code snippet shows how to scale the bounding box dimensions:
```

Comment:
Where in the code would this go?

Reply:
It wouldn't go anywhere since users aren't doing anything with it in the tutorial. I can add that this is for post-processing (if they choose to do so). Or we can expand the tutorial to include drawing the bounding boxes.